app/controllers/account_controllers.rb

```diff
@@ -0,0 +1,46 @@
+
+get '/accounts/login' do
+  erb :login
+end
+
+post '/accounts/login' do
+  user = User.find_by(email: params[:email])
+  db_password = user.password_hash
+  if db_password == params[:password]
```

Comment:
You should move this password check into a method on the user model. `user.authenticate(password)` is the convention.